CHANGELOG.md

```diff
@@ -3,6 +3,12 @@
 This article documents the ongoing improvements we're making to the **Cognite
 Data Source for Grafana**.
 
+## 4.3.0 - Oct 24th, 2024
+
+### Bug fixes
+
+- Upgrade plugin dependencies to fix CVEs
+
```

Comment:
have to bump `grafanaDependency` in [plugin.json](https://github.com/cognitedata/cognite-grafana-datasource/blob/master/src/plugin.json)
and make it as a breaking change